app/controllers/bus_stops_controller.rb

```diff
@@ -41,10 +41,11 @@ def id_search
   def manage
     @stops = BusStop.order(:name)
                     .paginate(page: params[:page], per_page: 10)
+    @all_stops = BusStop.order(:name)
```

Comment:
"all stops" doesn't _have_ to be an instance variable since there isn't a view

Reply:
Also, you could DRy this up a bit by doing it in the other order:

```ruby
all_stops = BusStop.order(:name)
@stops = all_stops.paginate(page: params[:page], per_page: 10)
```

Reply:
Oh, right. I will dry it. Thanks @werebus.